index.html

```diff
@@ -29,16 +30,14 @@
                 <div class="header">
                     <h2>Todo List</h2>
                 </div>
-    
-                <ul>
-                    <li>Todo 1</li>
-                    <li>Todo 2</li>
-                    <li>Todo 3</li>
-                    <li>Todo 4</li>
-                    <li>Todo 5</li>
-                </ul>
+                <form id="todoForm" onsubmit="createTodo()">
```

Comment:
I think we should use an event listener in the js file instead of calling the function in html, like you did with DOMContentLoaded

Reply:
Why I did add an eventlistner here is because its "local" call and that is what most seems to be the most common approach. And the eventlistner I implemented is "global", everytime we load the page we should load the users todos. 

Im not sure what the best practice is here?

Reply:
Yeah but you can use a "local" eventlistener for just the form.
```javascript
const form = document.getElementById("todoForm")

let createTodo = event => {...}
form.addEventListener("submit", createTodo)
```
Then the event listener is just listening for submits inside this specific form. 

But I'm not sure which is the best practice either. But as long as we are consistent then maybe someone will tell us one day :p so do whatever you please here, it's an easy fix later on if we are wrong